service/RTC/SignallingLayer.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
@saghul which one do you prefer ? I don't have strong opinion, but would leave it as is only, because I'm too lazy to change it...